/qrcode_detector:
  ros__parameters:
    # 单位mm
    qrcode_size: 100
    camera_matrix: [1167.922340402791, 0., 626.8356153449731, 0., 1171.400237544312, 383.593342116868, 0., 0., 1.]
    dist_coeffs:  [0.162794665656958, -0.3289701526905419, -0.0002266068165666849, 0.001158643714960362, 0.]
